kubernetes/linux/setup.sh

```diff
@@ -30,9 +30,9 @@ rm /usr/lib/ruby/gems/3.1.0/specifications/default/find-0.1.1.gemspec
 rm -rf /usr/lib/ruby/gems/3.1.0/gems/find-0.1.1
 
 if [ "${ARCH}" != "arm64" ]; then
-    wget "https://github.com/microsoft/Docker-Provider/releases/download/official%2Fmdsd%2F1.17.1%2Frpm/azure-mdsd_1.17.1-build.master.377_x86_64.rpm" -O azure-mdsd.rpm
+    wget "https://github.com/microsoft/Docker-Provider/releases/download/official%2Fmdsd%2F1.26.1/azure-mdsd-1.26.1-build.master.97.x86_64.rpm" -O azure-mdsd.rpm
```

Comment:
Does this version addresses all these 3 issues 1) IMDS token exception noise 2) syslog bug fix 3) syslog qos telemetry?

Reply:
1. Do not see any IMDS token exceptions in MSI mode
2. Syslog bug is already addressed in this build
3. Yes (eg MaODSRequest,   _LogManagement_LINUX_SYSLOGS_BLOB,15,15,0,147133,1268.27,0,19024,450,450)